test/WebSites/MvcTagHelpersWebSite/Views/MvcTagHelper_Home/Order.cshtml

```diff
@@ -33,6 +33,11 @@
             <select asp-for="@Model.Products" asp-items="@((IEnumerable<SelectListItem>)ViewBag.Items)" multiple="@(@object)" />
         </div>
         <div>
+            <label asp-for="SubstituteProducts" class="order" />
```

Comment:
nah, it's a fairly basic use of the helpers and this test already has the nice feature we have a parallel test exercising similar features using HTML tag helpers.